data/tmux.conf

```diff
@@ -5,20 +5,16 @@ bind -n F1 list-keys
 
 set-option -s exit-unattached off
 set-option -g base-index 1
-set-option -g set-remain-on-exit on
```

Comment:
tested on my laptop with tmux 2.2 and it works.
tested on a old server with tmux 1.8 and it does not (as in, it detaches on exit but I think it's because 1.8 does not support `set-remain-on-exit` :) )